common/src/androidTest/java/com/microsoft/identity/common/adal/internal/cache/StorageHelperTests.java

```diff
@@ -144,7 +144,7 @@ public void testEncryptDecryptDifferentSizes() throws GeneralSecurityException,
         // try different block sizes
         final int sizeRange = 1000;
         StringBuilder buf = new StringBuilder(sizeRange);
-        for (int i = 0; i < sizeRange; i++) {
+        for (int i = 0; i < sizeRange; i+=20) {
```

Comment:
Anything special about this value? Not clear to me

Reply:
Oh, the 20?  Not really.  1 at a time takes too long.